src/exml_nif.erl

```diff
@@ -11,11 +11,11 @@
 -type parser() :: term().
 -type stream_element() :: exml:element() | exml_stream:start() | exml_stream:stop().
 
--export([create/2, parse/1, parse_next/2, escape_cdata/1,
-         to_binary/2, reset_parser/1]).
+-export([create/2, parse/1, parse_next/2, escape_cdata/1, to_binary/2, reset_parser/1]).
+
 -export_type([parser/0, stream_element/0]).
 
--on_load(load/0).
+-on_load({load, 0}).
```

Comment:
Why? This looked completely reasonable.

Reply:
Because internally both are the same and the formatted is opinionated in this sense.

Reply:
Please open an issue on the rebar3_format repo to prefer the `name/#` syntax for `on_load` in the same way the formatter prefers that syntax for `-dialyzer` (I hope! 🤞).

Reply:
And this is the one for `on_load` - https://github.com/AdRoll/rebar3_format/issues/89